apis/contour/v1beta1/ingressroute.go

```diff
@@ -39,6 +39,8 @@ type VirtualHost struct {
 	// are described in fqdn and aliases, the tls.secretName secret must contain a
 	// matching certificate
 	TLS *TLS `json:"tls"`
+	// If set to false, this virtual host will only be accessible via HTTPS.
+	AllowHTTP *bool `json:"allowHttp"`
```

Comment:
Let's use the past tense here, s/AllowHTTP/HTTPAllowed/g here and everywhere to be consistent.